Tests/Common/Data/PsychSignalConsolidatorTests.cs

```diff
@@ -0,0 +1,530 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using NUnit.Framework;
+using QuantConnect.Data.Consolidators;
+using QuantConnect.Data.Custom.PsychSignal;
+
+namespace QuantConnect.Tests.Common.Data
+{
+    [TestFixture]
+    public class PsychSignalConsolidatorTests
+    {
+        [Test]
+        public void ConsolidatesMockPsychSignalDataAccurately_Daily()
+        {
+            // Note: This data is not real, and was created for testing purposes only
+            var consolidator = new PsychSignalConsolidator(TimeSpan.FromDays(1));
+            consolidator.DataConsolidated += (_, data) =>
+            {
+                Assert.AreEqual(new DateTime(2019, 1, 1), data.Time);
```

Comment:
Nice tests! 👍 
I think we could add an assertion on whether the callback `DataConsolidated `  was called or not at the end of each unit test
nit - might be more precise to move this file to ` Tests/Common/Data/Custom`

Reply:
RE: moving files to match namespaces -- completely agreed 👍